nf_core/components/components_utils.py

```diff
@@ -65,8 +53,30 @@ def get_repo_type(dir, repo_type=None, use_prompt=True):
     if not repo_type in ["pipeline", "modules"]:
         raise UserWarning(f"Invalid repository type: '{repo_type}'")
 
+    # Check for org if modules repo
+    org = None
+    if repo_type == "pipeline":
+        org = ""
+    elif repo_type == "modules":
```

Comment:
Hmm I think we do. Because of the use of `default_modules_path` (and the other related ones) which in a pipeline rep do have purpose but shouldn't have `org`. e.g. I think some parts of the code use it and expect`<pipeline_dir>/modules/`. I might be wrong though as I added this when I had a slightly different method to handle org.